resources/views/components/frameworks/bootstrap5/header/soft-deletes.blade.php

```diff
@@ -0,0 +1,29 @@
+@if(data_get($setUp, 'header.softDeletes'))
+    <div class="btn-group">
+        <button class="btn btn-secondary btn-sm dropdown-toggle"
+                type="button"
+                data-bs-toggle="dropdown"
+                aria-expanded="false">
+                    <span>
+                   <x-livewire-powergrid::icons.trash/>
+                </span>
+        </button>
+        <ul class="dropdown-menu">
+            <li wire:click="$emit('pg:softDeletes-{{ $tableName }}', '')">
+                <a class="dropdown-item" href="#">
+                    {{ __('--') }}
```

Comment:
Need to add insert translate for pt_Br and en for other translate

https://github.com/Power-Components/livewire-powergrid/blob/main/resources/lang/pt_BR/datatable.php

Example:
@lang('livewire-powergrid::datatable.with-trashed.all')